src/main/java/org/jabref/logic/integrity/FieldCheckers.java

```diff
@@ -53,6 +53,8 @@ private static Multimap<Field, ValueChecker> getAllMap(BibDatabaseContext databa
             fieldCheckers.put(StandardField.URLDATE, new DateChecker());
             fieldCheckers.put(StandardField.EVENTDATE, new DateChecker());
             fieldCheckers.put(StandardField.ORIGDATE, new DateChecker());
+//            fieldCheckers.put(StandardField.TITLE, new UnicodeNormalFormCCheck(databaseContext));
+//            fieldCheckers.put(StandardField.AUTHOR, new UnicodeNormalFormCCheck(databaseContext));
```

Comment:
Why did you remove the feld checkers? Are they now included in your new Unicode Normal Form Checker?
If so, please remove them completly, otherwise we have a lot of dead code fragments.

Reply:
Hi @calixtus 
No, these particular filechecker that I removed , is not included in new Unicode Normal Form Checker.
Initially, I was trying here but it was not correct. So, I removed

Reply:
But now they will remain commented. Will you uncomment these again or will you remove them?

Reply:
No, It will not uncomment again. 
We can completely remove them from here

Reply:
ok, then go ahead, remove them and all the dead code related to them. 👍

Reply:
Hi @calixtus 

I have removed that unnecessary comment from particular FieldCheckers file.  Now, It is good